composer.json

```diff
@@ -10,5 +10,10 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zendframework": "~2.5"
+    },
+    "autoload": {
+        "psr-4": {
+          "Application\\": "module/Application/src/Application"
```

Comment:
It works but the second Application folder is superfluous with PSR-4 so why have it?  Like I said: extreme pedantry.